salt/orchestrate/edx/init.sls

```diff
@@ -1,8 +1,8 @@
 {% from "orchestrate/aws_env_macro.jinja" import VPC_NAME, VPC_RESOURCE_SUFFIX,
  ENVIRONMENT, BUSINESS_UNIT, PURPOSE_PREFIX, subnet_ids with context %}
 
- {% set env_settings = salt.pillar.get('environment_settings') %}
- {% set bucket_prefixes = env_settings.edxapp_secret_backends.aws.bucket_prefixes %}
+{% set env_settings = salt.pillar.get('environments') %}
+{% set bucket_prefixes = edxapp_secret_backends.aws.bucket_prefixes %}
```

Comment:
This isn't directly accessible via the pillar file so you will need to dig it out of the nested structure.